net-misc/tigervnc/tigervnc-1.10.0.ebuild

```diff
@@ -0,0 +1,197 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools cmake-utils eutils flag-o-matic java-pkg-opt-2 systemd xdg-utils
```

Comment:
I didn't see *eutils.eclass* being used here, but this ebuild has a lot of text. Can you check is it used or not?